pkg/driver/driver.go

```diff
@@ -53,7 +53,8 @@ const (
 
 	WellKnownTopologyKey = "topology.kubernetes.io/zone"
 	// DEPRECATED Use the WellKnownTopologyKey instead
-	TopologyKey = "topology." + DriverName + "/zone"
+	TopologyKey   = "topology." + DriverName + "/zone"
```

Comment:
Do we want to rename this to `AZTopologyKey`?

Reply:
I can make this change for next rev